graphix_ibmq/runner.py

```diff
@@ -81,23 +81,45 @@ def to_qiskit(self, save_statevector=False):
         cr = ClassicalRegister(N_node)
         circ = QuantumCircuit(qr, cr)
 
-        empty_qubit = [i for i in range(n)]  # list indicating the free circuit qubits
+        empty_qubit = [i for i in range(n)]  # list indexicating the free circuit qubits
```

Comment:
just say `list of free qubit indices`?

Reply:
fixed this part. Thank you for pointing that out.